packages/mobile/src/fiatExchanges/CashInSuccess.tsx

```diff
@@ -14,19 +16,33 @@ import { StackParamList } from 'src/navigator/types'
 type RouteProps = StackScreenProps<StackParamList, Screens.CashInSuccess>
 type Props = RouteProps
 
+const capitalizeProvider = (provider?: string) => {
+  if (provider) {
+    const providerArr = provider.split('')
+    providerArr[0].toUpperCase()
```

Comment:
I forget now but doesn't this return a new string and so this might not do anything?

Reply:
What I'm doing is:
1. splitting every character into it's own array item
2. capitalizing the first item in the array
3. rejoining all the array items into a string

I've tested it out and think it works as expected :)